kinova_tracking/pos_kinova_arm.py

```diff
@@ -187,12 +90,12 @@ def prepare_ocp(
 
     for i, name in enumerate(markers_names):
         if name in labels_markers:
-            if name == 'Table:Table6':
-                markers[:, i, :] = points[:3, labels_markers.index('Table:Table5'), :] / 1000
+            if name == "Table:Table6":
+                markers[:, i, :] = points[:3, labels_markers.index("Table:Table5"), :] / 1000
             else:
                 markers[:, i, :] = points[:3, labels_markers.index(name), :] / 1000
```

Comment:
what is this ?

Reply:
To recreate the pivot we have to create an other marker just above the one which exist

Reply:
commented